jjb/dynamic/scale-ci_install_webfuse.yml

```diff
@@ -0,0 +1,246 @@
+- job:
+    block-downstream: false
+    block-upstream: false
+    builders:
+    - shell: |+
+        #!/bin/bash
+        set -o pipefail
+        set -eux
+
+        git clone https://${SSHKEY_TOKEN}@github.com/redhat-performance/perf-dept.git
+        export PUBLIC_KEY=${WORKSPACE}/perf-dept/ssh_keys/id_rsa_pbench_ec2.pub
+        export PRIVATE_KEY=${WORKSPACE}/perf-dept/ssh_keys/id_rsa_pbench_ec2
+        chmod 600 ${PRIVATE_KEY}
+
+        git clone https://${SSHKEY_TOKEN}@github.com/redhat-performance/J-Fleet
+        cp J-Fleet/ci/webfuse/group_vars/all.yml ansible/group_vars
+
+        pushd ansible
+        # Create inventory file
+        echo "[orchestration]" > hosts
+        echo "${ORCHESTRATION_HOST}   ansible_user=${ORCHESTRATION_USER}   ansible_ssh_private_key_file=${WORKSPACE}/perf-dept/ssh_keys/id_rsa_pbench_ec2" >> hosts
+
+        time ansible-playbook -i hosts ${WEBFUSE_PLAYBOOK} --tags ${WEBFUSE_TAGS} --skip-tags ${WEBFUSE_SKIPTAGS} | tee $(date +"%Y%m%d-%H%M%S")-jetski-install.timing
+    concurrent: false
+    description: |
+      Installs and configures VZW operators on the OCP cluster.
+      This job is managed by https://github.com/openshift-scale/scale-ci-pipeline
+    disabled: false
+    name: ATS-SCALE-CI-OCP-POST-CONFIG-WEBFUSE
+    node: baremetal-telco
+    parameters:
+    - string:
+        default: "kni"
+        description: The user for the Orchestration host.
+        name: ORCHESTRATION_USER
+    - string:
+        default: ""
+        description: Host with the kubeconfig and oc client.
+        name: ORCHESTRATION_HOST
+    - password:
+        default: ""
+        description: Token to access private repo containing ssh keys.
+        name: SSHKEY_TOKEN
+    - string:
+        default: "/home/kni/clusterconfigs/auth/config"
+        description: path to the kubeconfig
+        name: KUBECONFIG_PATH
+    - string:
+        default: ""
+        description: openshift release number to deploy
+        name: OPENSHIFT_RELEASE
+    - string:
+        default: "ga"
+        description: openshift build to deploy
+        name: OPENSHIFT_BUILD
+    - string:
+        default: "webscale.yml"
+        description: Playbook to install webfuse operators
+        name: WEBFUSE_PLAYBOOK
+    - string:
+        default: ""
+        description: Comma separated ansible tags 
+        name: WEBFUSE_TAGS
+    - string:
+        default: ""
+        description: Comma separated ansible tags
+        name: WEBFUSE_SKIPTAGS
+    - bool:
```

Comment:
Don't need this section anymore, lets remove OPENSHIFT_POST_CONFIG from the property file for this job and the deployment job. I left it in there for deployment job because I thought we'd use that flag to run webfuse, now we don't need this anymore.